extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -132,6 +123,12 @@ function _getDisplaySetsFromSeries(
   return [displaySet];
 }
 
+/**
+ * Loads the display set with the given services and extension manager.
+ * @param {Object} displaySet - The display set to load.
+ * @param {Object} servicesManager - The services manager containing displaySetService and measurementService.
+ * @param {Object} extensionManager - The extension manager containing data sources.
+ */
 async function _load(displaySet, servicesManager: AppTypes.ServicesManager, extensionManager) {
```

Comment:
There is a type definition for display set so that you can define both displaySet and extensionManager as the appropriate types.